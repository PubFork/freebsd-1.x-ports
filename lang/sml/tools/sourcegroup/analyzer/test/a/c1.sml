signature F  = sig
  val c :C.z
  include A         (* open A' *)
  val b :B.x
  val d :D.y
end
