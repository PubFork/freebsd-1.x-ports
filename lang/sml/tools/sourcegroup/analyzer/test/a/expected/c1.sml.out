source sml "a/c1.sml"
  import structure B C D 
  import signature A 
  export signature F ;

